Tests/Realm.Tests/Database/RealmValueWithCollections.cs

```diff
@@ -90,27 +90,14 @@ public void List_WhenRetrieved_WorksWithAllTypes([Values(true, false)] bool isMa
         }
 
         [Test]
-        public void List_InRealmValue_NotEqualToAnything([Values(true, false)] bool isManaged)
+        public void List_InRealmValue_Equality([Values(true, false)] bool isManaged)
         {
-            var innerList = ListGenerator(1);
-            var innerDict = DictGenerator(1);
-
             var originalList = new List<RealmValue>
             {
                 RealmValue.Null,
                 1,
                 true,
-                "string",
-                new byte[] { 0, 1, 2 },
-                new DateTimeOffset(1234, 5, 6, 7, 8, 9, TimeSpan.Zero),
-                1f,
-                2d,
-                3m,
-                new ObjectId("5f63e882536de46d71877979"),
-                Guid.Parse("3809d6d9-7618-4b3d-8044-2aa35fd02f31"),
-                new InternalObject { IntProperty = 10, StringProperty = "brown" },
-                innerList,
-                innerDict,
```

Comment:
Why the loss of cases?

Reply:
I could have kept the full list, but in the end it shouldn't make a difference on the test and I thought it would make the test easier to read. I didn't really need to 😁

Reply:
Okay.. gut reaction was just that the coverage was diminished, and would those tests now fail?

Reply:
That's understandable, I'll put those back so it's easier to follow